.github/workflows/test.yml

```diff
@@ -6,6 +6,7 @@ on:
   push:
     branches:
       - main
+      - release/**
```

Comment:
why was this added?

Reply:
artifact from other branch, will remove it.